app/views/application/_navigation.html.slim

```diff
@@ -24,6 +24,8 @@ nav.navbar.navbar-top.hidden-print
             li= link_to t_model(Department), department_path(current_user.department.first.id)
           - if policy(Assignment).index?
             li= link_to t('assignments'), assignments_path
+          - if policy(GroupOffer).index?
+            li= link_to t('.group_offers'), group_offers_path
```

Comment:
`t('.group_offers')` will return a different translation, depending on the actual controller. In that case it means you'd need to make sure that key .group_offers is available in any `controller_name.action_name.group_offers` key in the yaml files. Very likeley that this will fail.

Id do in the locale yaml:
```yaml
group_offers:
  one: Gruppenangebot
  other: Gruoppenangebote
```

```slim
li= link_to t('group_offers', count: 2), group_offers_path
```
Maybe also not perfect. Id Prefer t to have a Plural option. But unfortunately it doesn't, so that would be a work arround.

Reply:
t('.group_offers') looks for: [en/de].application.navigation.group_offers

Reply:
ok, wasn't aware of that